app/main.py

```diff
@@ -1,2 +1,51 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.hash_table = [[] for _ in range(self.capacity)]
```

Comment:
May better to create class `Node` and save (`key, value, hash`) as argument, and for empty cells use `None`

Reply:
what do you mean?